gli/io.py

```diff
@@ -1451,3 +1470,871 @@ def save_task_node_classification(name,
                                  num_samples=num_samples,
                                  task_id=task_id,
                                  save_dir=save_dir)
+
+
+def save_task_graph_regression(name,
+                               description,
+                               feature,
+                               target,
+                               train_set=None,
+                               val_set=None,
+                               test_set=None,
+                               train_ratio=0.8,
+                               val_ratio=0.1,
+                               test_ratio=0.1,
+                               num_samples=None,
+                               task_id=1,
+                               save_dir="."):
+    """Save the graph regression task information into task json and data files.
+
+    :param name: The name of the dataset.
+    :type name: str
+    :param description: The description of the task.
+    :type description: str
+    :param feature: The list of feature names to be used in the task. The
+        features could be node attributes, edge attributes, or graph
+        attributes. For homogeneous graphs, the feature names should be in the
+        format of "Node/{node_attr_name}", "Edge/{edge_attr_name}", or
+        "Graph/{graph_attr_name}". For heterogeneous graphs, the feature names
+        should be in the format of "Node/{node_type}/{node_attr_name}",
+        "Edge/{edge_type}/{edge_attr_name}", or
+        "Graph/{graph_type}/{graph_attr_name}". The node/edge/graph_attr_name,
+        and the node/edge/graph_type should be the ones declared in the
+        metadata.json file.
+    :type feature: list of str
+    :param target: The attribute name as prediction target in the task. For
+        this node regression task, the attribute should be a node attribute.
```

Comment:
"node regression" -> "graph regression". 

"should be a node attribute" - > "should be a graph attribute"

Please double check all the docstring